dlp/src/test/java/dlp/snippets/InspectTests.java

```diff
@@ -114,8 +121,13 @@ public void testInspectDatastoreEntity()
         PROJECT_ID, datastoreNamespace, datastoreKind, TOPIC_ID, SUBSCRIPTION_ID);
 
     String output = bout.toString();
-    assertThat(output, containsString("Info type: PHONE_NUMBER"));
-    assertThat(output, containsString("Info type: EMAIL_ADDRESS"));
+    assertThat(output, containsString("Job created: "));
+
+    String jobId = output.split("Job created: ")[1].split("\n")[0];
+    CancelDlpJobRequest request = CancelDlpJobRequest.newBuilder().setName(jobId).build();
+    try (DlpServiceClient client = DlpServiceClient.create()) {
+      client.cancelDlpJob(request);
+    }
```

Comment:
Fine if it works, but I'd bet these things can't happen right after the other.  (some hysteresis). 
(and it would be a flaky test again)